homework/Vasili_Zasinets/AbstractMath/src/test/java/com/tutrit/java/quickstart/abstractmath/DivisionOperationTest.java

```diff
@@ -0,0 +1,22 @@
+package com.tutrit.java.quickstart.abstractmath;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class DivisionOperationTest {
+
+    DivisionOperation divisionOperation = new DivisionOperation();
+
+    @Test
+    public void compute() throws DivisionByZeroException {
+        double expected = 0.8;
+        Assert.assertEquals(expected, divisionOperation.compute(4, 5), 0.000);
```

Comment:
1. use static import for Assert
2. "expected" instantiation redundant here.

Reply:
fixed